File: suwan/Programmers/step1/하샤드 수.py
def solution(arr):
    arr_sum = sum([int(i) for i in str(arr)])
    return arr % arr_sum == 0


arr1 = 10  # return true
arr2 = 12  # return true
arr3 = 11  # return false
arr4 = 13  # return false

print(solution(arr1))
print(solution(arr2))
print(solution(arr3))
print(solution(arr4))

File: seongkyu-lim/Programmers/JS/1step/하샤드수.js
function solution(x) {
    x = String(x)
    var x1 = 0;
    
    for(let i=0;i<x.length; i++){
        x1 += Number(x[i])
    }
    if (Number(x)%x1 == 0) {
        return true
    }
    else {
        return false
    }
}